content/opt/setup/icinga2

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+if [ ! "$(ls -A /etc/icinga2)" ]; then
+  echo "=> Copying fresh config-files for /etc/icinga2"
+  cp -R /etc/icinga2.dist/* /etc/icinga2/
+fi
+
+# chown directories and files that might be coming from volumes
+mkdir -p /var/log/icinga2/compat/archives
+chown -R nagios:adm /var/log/icinga2
+
+mkdir -p /var/lib/icinga2/api/zones
+mkdir -p /var/lib/icinga2/api/log
+mkdir -p /var/lib/icinga2/api/repository
+chown -R nagios:nagios /var/lib/icinga2
+
+chown -R nagios:root /etc/icinga2
+
+# enable neccessary features
+icinga2 feature enable ido-mysql livestatus compatlog command
+
+#icinga2 API cert - regenerate new private key and certificate when running in a new container
+if [ ! -f /etc/icinga2/pki/$(hostname).key ]; then
+	icinga2 node setup --master
+fi
+
+chfn -f "${ICINGA2_USER_FULLNAME}" nagios
+
+# MySQL installation
+mysql <<-END
+  CREATE DATABASE IF NOT EXISTS icinga;
+  GRANT SELECT, INSERT, UPDATE, DELETE, DROP, CREATE VIEW, INDEX, EXECUTE ON icinga.* TO 'icinga'@'localhost' IDENTIFIED BY '${ICINGA_PASSWORD}';
+END
+
+mysql icinga < /usr/share/icinga2-ido-mysql/schema/mysql.sql >> /var/log/icinga2/mysql-schema.log 2>&1
```

Comment:
Tested it: MySQL just stops after the first failing command, because there are duplicate columns in DB-Table. And first failing command is the first command, if the DB is already created and filled. So nothing bad will happen. I would "mark" this as done.